llvm/lib/Bitcode/Writer/BitcodeWriter.cpp

```diff
@@ -117,10 +118,6 @@ static cl::opt<bool>
 #endif
                                 cl::desc(""));
 
-namespace llvm {
-extern FunctionSummary::ForceSummaryHotnessType ForceSummaryEdgesCold;
```

Comment:
I suppose this could be simplified to "extern llvm::FunctionSummary::..." like the extern llvm::cl::opt below, if we decide to keep it.